src/components/question-editor/QuestionEditor.tsx

```diff
@@ -130,63 +156,108 @@ const QuestionEditor = () => {
     </Box>
   ))
 
+  const disabledButton =
+    type === 'openAnswer' ? text && openAnswer : text && answers[0]?.text
+
+  const buttons = (
+    <Box sx={styles.buttons}>
+      <AppButton onClick={() => navigate(-1)} variant={ButtonVariantEnum.Tonal}>
```

Comment:
maybe navigating to a certain URL will be safer?

Reply:
Changed